common/components/BalanceSidebar/EquivalentValues.tsx

```diff
@@ -197,8 +202,15 @@ export default class EquivalentValues extends React.Component<Props, CmpState> {
       return {};
     }
 
+    // Tokens with non-ether like decimals need to be adjusted to match
+    const decimal = this.decimalLookup[currency];
+    const adjustedBalance =
```

Comment:
is it possible to use functions inside `libs/units` instead of this?

Reply:
This now uses `toTokenBase`, however I think that I may not be using it as intended. I'm just using it to shift decimals, not to convert it to a particular tokens decimal. Perhaps the function should be renamed to be more general, or I should make a new function that makes it a little more clear what I'm doing, maybe something like `fromDecimalBaseToDecimalBase(value, decimalOld, decimalNew)`.

Reply:
Made a function `convertTokenBase` that does behavior we discussed in Slack. Also wrote unit tests for it. Should be cleaner now!